core/store/src/config.rs

```diff
@@ -37,12 +38,14 @@ pub struct StoreConfig {
     /// the performance of the storage
     pub block_size: bytesize::ByteSize,
 
-    /// Trie cache capacities
-    /// Default value: ShardUId {version: 1, shard_id: 3} -> 45_000_000
-    /// We're still experimenting with this parameter and it seems decreasing its value can improve
-    /// the performance of the storage
+    /// DEPRECATED: use `trie_cache` instead.
```

Comment:
When adding a derecation comments, it's a good idea to mention the future release when the stuff should be removed. That way, someone from the future can easily see that the thing can *already* be removed.